src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,35 @@
-export const PostInfo = () => <>Put the post here</>;
+import './PostInfo.scss';
+
+import usersFromServer from '../../api/users.json';
+
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+const getUserById = userId =>
+  usersFromServer.find(user => user.id === userId) || null;
+
+export const PostInfo = ({ post }) => {
+  const user = getUserById(post.userId);
```

Comment:
Representation components should accept data and render it without the logic to find something